app/services/current-session.js

```diff
@@ -9,13 +9,17 @@ export default class CurrentSessionService extends Service {
   @tracked user;
   @tracked group;
   @tracked roles;
+  @tracked organization;
 
   async load() {
     if (this.session.isAuthenticated) {
       const accountId = this.session.get('data.authenticated.relationships.account.data.id');
       this.account = await this.store.find('account', accountId);
       this.user = await this.account.user;
 
+      const groupUri = await this.user.group.get('uri');
```

Comment:
Je kan dit beste verder naar onder verhuizen nadat `this.group` al ingesteld is. Dan kan je gewoon `this.group.uri` gebruiken zonder de `await` en `get()` omdat de group al opgehaald is uit de backend.

Reply:
ik heb het ophalen van de organization in de if gezet, omdat we dan zeker zijn dat er een group is